code/datums/traits/negative.dm

```diff
@@ -53,17 +53,48 @@
 		H.put_in_hands(B)
 	H.regenerate_icons()
 
+	/* A couple of brain tumor stats for anyone curious / looking at this quirk for balancing:
+	 * - It takes less 16 minute 40 seconds to die from brain death due to a brain tumor.
+	 * - It takes 1 minutes 40 seconds to take 10% (20 organ damage) brain damage.
+	 * - 5u mannitol will heal 12.5% (25 organ damage) brain damage
+	 */
 /datum/quirk/brainproblems
 	name = "Brain Tumor"
 	desc = "You have a little friend in your brain that is slowly destroying it. Better bring some mannitol!"
 	value = -3
+	var/where
 	gain_text = "<span class='danger'>You feel smooth.</span>"
 	lose_text = "<span class='notice'>You feel wrinkled again.</span>"
 	medical_record_text = "Patient has a tumor in their brain that is slowly driving them to brain death."
 	hardcore_value = 12
 
+/datum/quirk/brainproblems/on_spawn()
+	var/mob/living/carbon/human/H = quirk_holder
+	var/pills = new /obj/item/storage/pill_bottle/mannitol/braintumor()
+	var/list/slots = list(
+		"in your left pocket" = ITEM_SLOT_LPOCKET,
+		"in your right pocket" = ITEM_SLOT_RPOCKET,
+		"in your backpack" = ITEM_SLOT_BACKPACK,
+		"in your hands" = ITEM_SLOT_HANDS
+	)
+	where = H.equip_in_one_of_slots(pills, slots, FALSE) || "at your feet"
```

Comment:
|| compares so it will either return 0/1

Reply:
the other traits that spawn the player with items use the same line, so I assume it works?